packages/react-components/react-combobox/stories/src/Combobox/ComboboxControllingOpenAndClose.stories.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import { Checkbox, Combobox, makeStyles, Option, useId } from '@fluentui/react-components';
+import type { CheckboxProps, ComboboxProps } from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+  root: {
+    // Stack the label above the field with a gap
+    display: 'grid',
+    gridTemplateRows: 'repeat(1fr)',
+    justifyItems: 'start',
+    gap: '2px',
+    maxWidth: '400px',
+  },
+});
+
+export const ControllingOpenAndClose = (props: Partial<ComboboxProps>) => {
+  const comboId = useId('combo-default');
+  const options = ['Cat', 'Dog', 'Ferret', 'Fish', 'Hamster', 'Snake'];
+  const styles = useStyles();
+
+  const ref = React.useRef<HTMLInputElement>(null);
+  const [open, setOpen] = React.useState(false);
+  const handleOpenChange: ComboboxProps['onOpenChange'] = (e, data) => setOpen(data.open || false);
+
+  const onChange: CheckboxProps['onChange'] = (e, { checked }) => {
+    const isOpen = Boolean(checked) || false;
+    setOpen(isOpen);
+
+    if (isOpen) {
+      ref.current?.focus();
+    } else {
+      ref.current?.blur();
+    }
+  };
+
+  return (
+    <div className={styles.root}>
+      <label id={comboId}>Best pet</label>
+      <Combobox
+        aria-labelledby={comboId}
+        placeholder="Select an animal"
+        ref={ref}
+        open={open}
+        onOpenChange={handleOpenChange}
+      >
+        {options.map(option => (
+          <Option key={option} disabled={option === 'Ferret'}>
+            {option}
+          </Option>
+        ))}
+      </Combobox>
+      <Checkbox value="open" name="state" label="open" checked={open} onChange={onChange} />
```

Comment:
I'd move the checkbox above or to the side of the combobox since it's immediately covered when opening for me
![combobox covering](https://github.com/microsoft/fluentui/assets/20744592/e070b75e-498a-4d0d-8bb1-fee08998bbe7)

Reply:
Updated